src/index.ts

```diff
@@ -0,0 +1,21 @@
+import marked from "marked"
+import babel from "@babel/core"
+
+export async function parse(markdown: string) {
+  const snippets = marked
+    .lexer(markdown)
+    .filter((token) => token.type === "code" && token.lang === "js")
+    .map((token) => token.raw)
```

Comment:
Property `text` doesn't exist in the `Token` type. Is this typings mismatch, or an issue?